src/window/Window.cpp

```diff
@@ -45,8 +45,7 @@ Window::~Window() {
 }
 
 void Window::Init() {
-    bool steamDeckGameMode = false;
-    bool androidGameMode = false;
+    bool mGameMode = false;
```

Comment:
out of scope for this PR and also mostly a @Kenix3 question but this has always felt a bit like a kitchen sink variable to me

 i'm thinking we should make an issue to find all usages of `mGameMode` and figure out either
* what a good name for it would be
or
* if we want to split it up into multiple variables to handle different things it changes

Reply:
https://github.com/Kenix3/libultraship/issues/525